pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -54,7 +54,7 @@ func NewStorageFactoryConfig() *StorageFactoryConfig {
 		DefaultResourceEncoding: serverstorage.NewDefaultResourceEncodingConfig(legacyscheme.Scheme),
 		ResourceEncodingOverrides: []schema.GroupVersionResource{
 			batch.Resource("cronjobs").WithVersion("v1beta1"),
-			apisstorage.Resource("volumeattachments").WithVersion("v1beta1"),
+			apisstorage.Resource("volumeattachments").WithVersion("v1"),
```

Comment:
Curious if this is still needed

Reply:
v1 is the preferred version, you can drop this line completely